dbms/programs/server/config.xml

```diff
@@ -35,40 +35,12 @@
     <http_port>8123</http_port>
     <tcp_port>9000</tcp_port>
     <mysql_port>9004</mysql_port>
-    <!-- For HTTPS and SSL over native protocol. -->
+    <!-- For HTTPS and SSL over native protocol. See also ssl.xml in config.d/-->
```

Comment:
config.xml is unstalled in clickhouse-server package but files from config.d are not.